include/mcl_3dl/lidar_measurement_model_base.h

```diff
@@ -64,6 +65,12 @@ class LidarMeasurementModelBase
 public:
   using Ptr = std::shared_ptr<LidarMeasurementModelBase>;
   using PointType = mcl_3dl::PointXYZIL;
+  using SamplerType = PointCloudRandomSampler<PointType>;
+
+  LidarMeasurementModelBase()
+    : sampler_()
```

Comment:
I think it's better to be set by a constructor argument (from derived class constructor) as it will cause null pointer error if `setRandomSampler` is not called.

Reply:
Or, it can be passed as an argument of `LidarMeasurementModel::filter()`.

Reply:
PointCloudUniformSampler is set as a default.